cmd/ejson/main.go

```diff
@@ -65,9 +66,22 @@ func main() {
 					Name:  "o",
 					Usage: "print output to the provided file, rather than stdout",
 				},
+				cli.BoolTFlag{
+					Name:  "key-from-stdin",
+					Usage: "Read the private key from STDIN",
+				},
 			},
 			Action: func(c *cli.Context) {
-				if err := decryptAction(c.Args(), c.GlobalString("keydir"), c.String("o")); err != nil {
+				var userSuppliedPrivateKey string
+				if c.Bool("key-from-stdin") {
+					stdinContent, err := ioutil.ReadAll(os.Stdin)
+					if err != nil {
+						fmt.Println("Failed to read from stdin:", err)
+						os.Exit(1)
+					}
+					userSuppliedPrivateKey = string(stdinContent)
```

Comment:
This is probably a good place to sanitize it -- at least trim the likely trailing newline, maybe with `strings.TrimSpace`